src/content/config.ts

```diff
@@ -10,8 +10,8 @@ const termsCollection = defineCollection({
     description: z.object({
       title: z.string().min(1),
       texts: z.array(z.string()).nonempty(),
-      image: z.string().optional(),
-      references: z.array(z.string()).nonempty()
+      image: z.union([z.string().url().nullish(), z.literal("")]),
```

Comment:
> `z.union([z.string().url().nullish(), z.literal("")])`

Do we need `nullish` here? We are already using a union which means an image can either be a `UrlString` or `""` empty literal. I just did a test [here](https://zod-playground.vercel.app?appdata=N4IgzgxgFgpgtgQxALhALwHQHsBGArGCAFwApgAdAOwAJqB3ASyKgDkBXAGw4bCmWswIATkIQBPEpjaUGWSiQDamMESENKAcxIBKDGyEcdGSp268dAGgEZuRGKMPkQT7QF1t2i1VqU57Ljx81sKiElIycorKqupauvqGnta29giOziBuHl6UAL7aIBYgAG5pbDBgKAogFELetNROjMz%2BZlBO-AomXFbSACYwAGbqMH1WTk7jIFBERAAOYMgA9EtoWH0AtHMc4hpCWP0YxfYQMBwYCHNzS06uXnU0DU6%2BlK2BHdQKE4WN07MLy1W6y2OzEewOlD6RxOZwuVxuIFc5AeuURRWOQjAskoKBAAGYMAAmAkADhAuSAA), and my assumption is correct.

On a separate note, should we really allow empty non-nullish properties - like image? It doesn't make sense for a property to exist in the first place if it has to be empty.